### ISSUE TITLE
macOS Mojave: Safe to remove?

### ISSUE THREAD

**Issue by NPCtom** [zakk4223/SyphonInject#10]
I installed OBS on macOS Mojave and it still works. I dont know what SyphonInject is used for but I was wondering is it safe to remove as it is annoying showing up in my applications folder.

**Comment by zakk4223** [comment]
I don't believe OBS uses the actual SyphonInject app, but you would have to ask them.

**Comment by HasanChute** [comment]
I have a same issue could you please tell if it is safe to delete it. by the way what the use for it I only want the obs

**Comment by zakk4223** [comment]
OBS bundles SyphonInject for one or two of their capture sources. I don't know if they actually use the SyphonInject.app or not. Please ask the OBS forums/support these questions
